strawberryfields/apps/qchem/__init__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains functionality for quantum chemistry calculations.
```

Comment:
Note I added this `__init__.py`, we can update the sentences here as the module evolves (if we need to).

Reply:
Noted, thanks.